docs/concepts/privacy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Privacy
+sidebarDepth: 0
+description: Learn about user privacy in IPFS and why it does not come with a built-in privacy layer or encryption.
+related:
+  'Article: IPFS Privacy (Textile)': https://medium.com/pinata/ipfs-privacy-711f4b72b2ea
+---
+
+# IPFS and privacy
+
+As you explore IPFS and learn about its core concepts, it's important to know that as a protocol for peer-to-peer data storage and delivery, IPFS in itself does not explicitly have a "privacy layer" built-in. This is in line with key principles of protocol design since different uses of IPFS over its life may call for different approaches to privacy. Explicitly implementing an approach to privacy within the IPFS core could "box in" future builders and implementers due to a lack of modularity, flexibility, and future-proofing.
+
+By design, IPFS is a public network: Nodes participating in the network store data affiliated with globally consistent [content addresses](/concepts/content-addressing) (CIDs) and advertise that they have those CIDs available for other nodes to use through publicly viewable [distributed hash tables](/concepts/dht/) (DHTs). Indeed, this is one of IPFS's core strengths — at its most basic, it's a sort of globally distributed "server" of the network's total available data, referenceable both by the content itself (those CIDs) and by the participants who have or want the content (the nodes).
```

Comment:
confusing to have "(the nodes)" after "the content", should move to after "participants"?

Reply:
s/sort of// or something clearer.

Reply:
https://github.com/ipfs/ipfs-docs/pull/554/commits/6ed4faa9df2826b17d8964b49faf63e2fbabc582 should take care of both of these